cpp/velox/memory/VeloxMemoryManager.cc

```diff
@@ -280,6 +284,9 @@ VeloxMemoryManager::~VeloxMemoryManager() {
     usleep(waitMs * 1000);
     accumulatedWaitMs += waitMs;
   }
+#ifdef ENABLE_JEMALLOC
+  je_gluten_malloc_stats_print(NULL, NULL, NULL);
```

Comment:
It printed too many logs if we put here. Every task will print a log.

@PHILO-HE  @zhztheplayer Does Gluten has callback of each query? If so let's put the dump to the callback. if not see if we can add one.

Reply:
@zhztheplayer, as mentioned in the meeting, could you provide some details about how CH backend does similarly when query execution is exiting?

Reply:
Hi, please take a loot at this PR  https://github.com/oap-project/gluten/pull/1604 to see if the facility it added could help.